### ISSUE TITLE
Add author type on the image's informations

### ISSUE THREAD

**Issue by flob38** [c2corg/c2c_ui#488]
In the available informations for images, we have 2 sort of author : author of the document (the one who created the document on C2C) and the author of the image (the one who shooted it).
When displaying those info, we must use suitable labels.

![image](https://user-images.githubusercontent.com/24532066/55711112-5aadbc00-59ec-11e9-938a-35f551298a50.png)

**Comment by flob38** [comment]
Maybe it is only a problem of traduction. 
```
<field-view :document="document" :field="fields.author" />
<label-value v-if="document.creator" :label="$gettext('creator')">
     <author-link :author="document.creator" />
</label-value>
```

**Comment by cbeauchesne** [comment]
First line : it's image author, which means the one who took the picture
Second line : it's the one who uploaded the picture on camptocamp.

We must ask community what are the best translations for them, and fix it on transifex.

I propose : 

* first line : Auteur de l'image
* second line : Contributeur

Note that the first translation key is fixed, as it comes from API. It will be `author`. 
The second translation key, actually `creator` can be changed, and as it's actually missleading, must be changed. If we choose "contributeur" in french, we'll change translation key to `contributor`.

**Comment by cbeauchesne** [comment]
Duplicated of #590, and it's fixed by #591. Sorry @flob38, I've forgotten that you have created this issue when I opened #590
